frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I'm pretty sure projects normally only have one package.json which is typically located at the root

Reply:
Yeah that makes sense. I'll look into it and make an update.

Reply:
Since we're running one server on port 3000 and one on port 5000, as far as I can tell we need 2 projects to run concurrently. I could be wrong though.

Reply:
This will be fixed once it is hosted somewhere else than localhost I believe